packages/Organisms/CampaignPrimary/index.vue

```diff
@@ -112,6 +112,10 @@ export default {
     }
```

Comment:
Is there any reason this logo issue need to change `Campaign Primary` component?